.github/workflows/ci.yml

```diff
@@ -206,22 +207,33 @@ jobs:
         make all || true
         python -m pip install --upgrade pip
         python -m pip install pytest
-        python -m pytest --setup-plan | grep -o -P '(?<=test\/).*(?=\ \(fixtures)' | \
-          while read line ; do
-            NO_OF_TESTS=$((NO_OF_TESTS+1))
-            python -m pytest -s $line >> test.log 2>&1
-            if [[ $? -eq 0 ]]; then
-               PASSED=$((PASSED+1))
-               echo "$line PASSED"
-            else
-               echo "$line FAILED"
-            fi
-         done
+        export PASSED=0
+        export NO_OF_TESTS=0
+        export SETUP_PLAN=$(python -m pytest --setup-plan | grep -o -P '(?<=test\/).*?(?=\ \(fixtures)')
+        echo ::group::Test Status
+        set +e
+        for test in ${SETUP_PLAN} ; do
+          NO_OF_TESTS=$((NO_OF_TESTS+1))
+          (
+            python -m pytest -s $test >> test.log 2>&1
+          )
+          if [[ $? -eq 0 ]]; then
+            PASSED=$((PASSED+1))
+            echo "$test PASSED"
+          else
+            echo "$test FAILED"
+          fi
+        done
+        set -e
+        echo ::endgroup::
```

Comment:
Can we print a summary of the passed vs failed tests here?

Reply:
Could you elaborate?

Reply:
Printing something like:
Tests Passed: N
Tests Failed: M